README.md

```diff
@@ -34,11 +34,21 @@ Feel free to try out and let us know if you cannot wait.
 
 ## Install dockert3kit into your distribution
 
-Add `lauri/dockert3kit` as dev dependency in your composer, using the latest stable release is highly recommended.
+Add `lauri/dockert3kit` as dev dependency in your composer.
+
+***Note***
+
+**If you are using EXT:solr = 6.0.0 please use the 6.0.0 release**
+
+*Example*:
+
+    composer require --dev lauri/dockert3kit '~6.0.0'
+
+**If you are using EXT:solr = 6.1.0 please use the 6.1.0 release**
 
 *Example*:
 
-    composer require --dev lauri/dockert3kit '~3.0.0'
+    composer require --dev lauri/dockert3kit '~6.1.0'
```

Comment:
@laurisaarni ok, so how should be name the release? It would have been easier to remember and track if we match the release with the ext-solr version.